test/grpc/route_guide/record-route.cc

```diff
@@ -0,0 +1,65 @@
+#include <string>
+
+#include "eventuals/finally.h"
+#include "eventuals/foreach.h"
+#include "eventuals/head.h"
+#include "eventuals/iterate.h"
+#include "eventuals/let.h"
+#include "eventuals/promisify.h"
+#include "eventuals/range.h"
+#include "eventuals/then.h"
+#include "gtest/gtest.h"
+#include "test/grpc/route_guide/route-guide-eventuals-test.h"
+
+using eventuals::grpc::Stream;
+using grpc::Status;
+
+using eventuals::Finally;
+using eventuals::Foreach;
+using eventuals::operator*;
+using eventuals::Head;
+using eventuals::Iterate;
+using eventuals::Let;
+using eventuals::Range;
+using eventuals::Then;
+
+using routeguide::Point;
+
+TEST_F(RouteGuideTest, RecordRouteTest) {
+  std::vector<Point> data(GetPointsCount());
+
+  for (size_t i = 0; i < GetPointsCount(); ++i) {
+    data[i] = feature_list_[i].location();
+  }
+
+  auto e = [&]() {
+    return client->RecordRoute()
+        >> Then(Let([&data](auto& call) mutable {
+             return Foreach(
+                        Iterate(std::move(data)),
+                        ([&call](auto&& input) {
```

Comment:
These are requests, please rename all `input` to `request` and all `output` to `response`.